### DISCUSSION TITLE
cxf component- as service/consumer

### DISCUSSION THREAD

**Discussion by manali-jambhale** [apache/camel-karavan#759] [General]
When i am trying to use cxf component as the starting step of a route getting the following  error

org.apache.camel.PropertyBindingException: Error binding property (camel.component.rest.consumerComponentName=platform-http) with name: consumerComponentName on bean: org.apache.camel.component.rest.RestComponent@13741d5a with value: platform-http

Can you suggest how to resolve this?..or provide any example route

**Comment by davsclaus** [comment]
Can you maybe provide a sample project that reproduces this. At first glance it seems you shomehow run Camel without having a HTTP server included.
